projects/packages/forms/src/modules/slider-field/view.js

```diff
@@ -0,0 +1,44 @@
+import { store, getContext } from '@wordpress/interactivity';
+
+const NAMESPACE = 'jetpack/form';
+
+/**
+ * Gets the min and max values from the slider input element within the current context node.
+ *
+ * @param {object} context - The interactivity context for the current slider field.
+ * @return {{min: number, max: number}} The min and max values for the slider.
+ */
+function getSliderMinMax( context ) {
+	const root = context._rootNode || document;
+	const input = root.querySelector( 'input[type="range"]' );
```

Comment:
instead of querying the dom could we not pass in the min and max value via the context? 
via `data-wp-context='{ "foo": "bar" }' ` see https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/#wp-context

Reply:
Yep, I've updated this!